pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaConfigBackwardCompatibleUtils.java

```diff
@@ -35,11 +35,13 @@ private KafkaConfigBackwardCompatibleUtils() {
    */
   public static void handleStreamConfig(StreamConfig streamConfig) {
     Map<String, String> streamConfigMap = streamConfig.getStreamConfigsMap();
+    //FIXME: This needs to be done because maven shade plugin also overwrites the constants in the classes
+    String prefixToReplace = KAFKA_COMMON_PACKAGE_PREFIX.replace(PINOT_SHADED_PACKAGE_PREFIX, "");
```

Comment:
Just to give readers some context, this is what `javap` shows for this line before shading:
```
      36: ldc           #9                  // String org.apache.kafka.common
      38: ldc           #10                 // String org.apache.pinot.shaded.
      40: ldc           #11                 // String
      42: invokevirtual #12                 // Method java/lang/String.replace:(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
```

And this is what it shows after shading:
```
      36: ldc           #19                 // String org.apache.pinot.shaded.org.apache.kafka.common
      38: ldc           #22                 // String org.apache.pinot.shaded.
      40: ldc           #60                 // String
      42: invokevirtual #66                 // Method java/lang/String.replace:(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Ljava/lang/String;
```